forui/lib/src/widgets/select_group/select_group_controller.dart

```diff
@@ -29,12 +30,64 @@ abstract class FSelectGroupController<T> extends FChangeNotifier {
   @override
   bool operator ==(Object other) =>
       identical(this, other) ||
-      other is FSelectGroupController && runtimeType == other.runtimeType && _values == other._values;
+      other is FSelectGroupController && runtimeType == other.runtimeType && values == other.values;
 
   @override
   int get hashCode => _values.hashCode;
 }
 
+@internal
+// ignore: avoid_implementing_value_types
+class DelegateSelectGroupController<T> implements FSelectGroupController<T> {
```

Comment:
What's this for?

Reply:
It's used in `FSelectTileMenu` to enable the `autoHide` feature.